ifacemonitor/ifacemonitor_test.go

```diff
@@ -280,7 +292,14 @@ func (dp *mockDataplane) addrStateCallback(ifaceName string, addrs set.Set) {
 }
 
 func (dp *mockDataplane) notExpectAddrStateCb() {
-	Consistently(dp.addrC).ShouldNot(Receive())
+	Consistently(func() bool {
+		select {
+		case <-dp.addrC:
+			return true
+		default:
+			return false
+		}
+	}, "1s", "200ms").ShouldNot(BeTrue())
```

Comment:
Does `Consistently(dp.addrC, "1s", "200ms").ShouldNot(Receive())` not work?  (I didn't think that you would need a change as big as this.)